src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpInterop.cs

```diff
@@ -69,8 +69,7 @@ public static void GetResponseHeaders(JSObject fetchResponse, HttpHeaders respos
             string[] headerValues,
             string[] optionNames,
             [JSMarshalAs<JSType.Array<JSType.Any>>] object?[] optionValues,
-            JSObject abortControler,
-            string? body = null);
+            JSObject abortControler);
```

Comment:
I think this is now un-used, no ?

Reply:
It is needed for requests without content like `GET`.